### PR TITLE
Fix/remove lodash

### LINKED ISSUE #406: Remove lodash dep

### LINKED ISSUE #420: Add an editor config
**Describe the bug**
Currently there is no .editorconfig. Beneficial for standardizing open source contribution.

### PR DESCRIPTION
**Pull request by mattbford**
fixes #406
fixes #420

### CODE REVIEW

File: src/method-call-validator/method-call-validator.ts [hunk 1]
```diff
@@ -64,28 +64,28 @@ export default class MethodCallValidator {
...
+        
         if (method.paramStructure === "by-position") {
           id = index;
         } else if (method.paramStructure === "by-name") {
           id = param.name;
         } else {
           if (params[index] !== undefined ) {
             id = index;
-          } else {
+          } else {
```
Comment (shanejonas) [inline src/method-call-validator/method-call-validator.ts:85]: some formatting issues, i think this repo is missing the `.editorconfig` maybe?

Comment (mattbford) [reply src/method-call-validator/method-call-validator.ts]: added the config

File: src/helper-functions.ts [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+};
+
+/**
+ * compares OpenrpcDocuments
+ *
+ * @param doc1 {OpenrpcDocument}
+ * @param doc2 {OpenrpcDocument}
+ * @returns {boolean}
+*/
+export const rpcDocIsEqual = (doc1: any, doc2: any) => {
```
Comment (BelfordZ) [inline src/helper-functions.ts:65]: If it's only included in the test, you can put it in a `.test.` file to have it not included in the build.

Comment (BelfordZ) [reply src/helper-functions.ts:65]: https://github.com/open-rpc/schema-utils-js/blob/master/package.json#L17

**Pull request merged**